java.java

```diff
@@ -0,0 +1,11 @@
+// pacage descrition
```

Comment:
Typo inside comment in word `pacage`. Possible corrections: package (35), packages (0), manage (0),
Typo inside comment in word `descrition`. Possible corrections: description (98), descrition (0),